src/convertToObject.js

```diff
@@ -12,7 +12,16 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString.split(';');
```

Comment:
What if you will have another array here? It will be `array2`?
Variable name should describe content inside, not type.